internal/rbd/encryption.go

```diff
@@ -341,8 +341,14 @@ func ParseEncryptionOpts(
 		encrypted, kmsID string
 	)
 	encrypted, ok = volOptions["encrypted"]
-	val, _ := strconv.ParseBool(encrypted)
-	if !ok || !val{
+	if !ok {
+		return "", util.EncryptionTypeNone, nil
+	}
+	val, er := strconv.ParseBool(encrypted)
```

Comment:
you can re-use both `ok` and `err` variables here. No need to introduce two new ones.